### PR TITLE
Updated to building openssl ourselves

### PR DESCRIPTION
**Pull request by kbarkhqs**

### CODE REVIEW

File: qoqo_aqt/docs/requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+qoqo_calculator_pyo3>=1.1.0
 numpy
 sphinx >= 2.1
 nbsphinx
 pygments
 recommonmark
 myst_parser
 sphinx_rtd_theme
-qoqo_aqt>=0.5.1
+qoqo_aqt>=0.6.1
```
Comment (mlodi-hqs) [inline qoqo_aqt/docs/requirements.txt:10]: Sticking to `1.2`, `1.1` and `0.6` should be enough if you want.

File: roqoqo-aqt/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+For other platforms we recommend checking out the latest tagged version from github and using the [maturin](https://github.com/PyO3/maturin) tool to build a python package for qoqo locally and install it via pip.
+Please note that the package should be built from the top level directory of the workspace selecting the qoqo package with the `-m qoqo/Cargo.toml` option.
+Specifically for macOS on Apple Silicon the following build command should be used.
+
+```shell
+RUSTFLAGS="-C link-arg=-undefined -C link-arg=dynamic_lookup" maturin build -m qoqo_aqt/Cargo.toml  --release
+pip install target/wheels/$NAME_OF_WHEEL
+```
+
+A source distribution now exists but requires a Rust install with a rust version > 1.47 and a maturin version { >= 0.12, <0.13 } in order to be built.
```
Comment (mlodi-hqs) [inline roqoqo-aqt/README.md:48]: Is this correct? Shouldn't it be >=0.14 <0.15?

Comment (mlodi-hqs) [reply roqoqo-aqt/README.md:48]: And Rust 1.56 I guess

**Review by mlodi-hqs — APPROVED** [review]

File: qoqo_aqt/docs/requirements.txt (hunk 1 above)
Comment (kbarkhqs) [reply qoqo_aqt/docs/requirements.txt:10]: Thank you!

File: qoqo_aqt/qoqo_aqt/LICENSE_FOR_BINARY_DISTRIBUTION [hunk 3]
```diff
@@ -6,6 +6,8 @@ This product in binary form links dynamically against Python  available under th
 Python (Copyright © 2001-2023 Python Software Foundation; All Rights
 Reserved)
 
+This product in binary form links against OpenSSL. Depending on the linked version of OpenSSL the dual OpenSSL and SSLeay license applies (see file OPENSSL_LICENSE) or the Apache-2.0 license applies (since version 3.0).
```
Comment (nfwvogt) [inline qoqo_aqt/qoqo_aqt/LICENSE_FOR_BINARY_DISTRIBUTION:9]: We still need to add the OPENSSL_LICENSE file

**Review by nfwvogt — CHANGES_REQUESTED** [review]
Just one missing license file

**Review by nfwvogt — APPROVED** [review]

**Pull request merged**